src/gateway/handlers/make-render-handler.js

```diff
@@ -39,11 +39,15 @@ async function renderHandler(
     const traceFn = (name: string): ITraceSession => trace(name, req);
 
     /**
-     * TODO(somewhatabstract, WEB-1856): Currently passing the entire URL, but
-     * we want to be more specific here and define the render route better as
-     * we'll really want an absolute URL.
+     * The URL being rendered is given in a query param named, url.
      */
-    const renderURL = req.url;
+    const renderURL = req.query["url"];
```

Comment:
Can just be: `req.query.url`

Reply:
I was following the express examples about this that seemed to favor the indexing approach.